packages/components/src/notice/style.scss

```diff
@@ -1,4 +1,6 @@
 .components-notice {
+	font-family: $default-font;
+	font-size: $default-font-size;
```

Comment:
This makes sense to me. I wonder if we should do an audit and do this consistently across our reusable UI components. For example, I also noticed that the block placeholders and the buttons can be altered by editor styles.

Also what properties should we add: `font-family`, `font-size`, maybe also `color`, `background` and `line-height`?

Reply:
Yeah, definitely might be worth an audit. 👍 I'll merge this in for now though.